trade-finance-manager-ui/templates/case/parties/_macros/parties-key-value-grid-row.njk

```diff
@@ -9,10 +9,14 @@
         {% endif %}
     </div>
     <div class="govuk-grid-column-two-thirds">
-      {% if params.dataCy | safe %}<p class="govuk-body-s govuk-!-margin-top-3 {{ params.valueStyle }}" data-cy="{{ params.dataCy }}">{% else %}<p class="govuk-body-s {{ params.valueStyle }}">{% endif %}
-        {{ params.value }}
+      {% if params.dataCy | safe %}
+        <p class="govuk-body-s govuk-!-margin-top-3 {{ params.valueStyle }}" data-cy="{{ params.dataCy }}">
```

Comment:
Why is there a top margin if dataCy exists?

Reply:
Honestly I don't know, the macros in this section of TFM are not very clear. I didn't add or modify this part, I just changed the indentation so I could read it and add a "dashIfEmpty" nunjucks filter 🙃